examples/parallel/distributed_backend_simple.py

```diff
@@ -0,0 +1,57 @@
+"""
+Using distributed for single_machine parallel computing
+========================================================
+
+This example shows the simplest usage of the dask `distributed
+<https://distributed.readthedocs.io>`__ backend, on the local computer.
+
+This is useful for prototyping a solution, to later be run on a truly
+distributed cluster, as the only change to be made is the address of the
+scheduler.
+
+Another realistic usage scenario: combining dask code with joblib code,
+for instance using dask for preprocessing data, and scikit-learn for
+machine learning. In such a setting, it may be interesting to use
+distributed as a backend scheduler for both dask and joblib, to
+orchestrate well the computation.
+
+"""
+
+###############################################################################
+# Setup the distributed client
+###############################################################################
+from distributed import Client
```

Comment:
Similarly we tend to encourage `from dask.distributed import Client` in examples

Reply:
This is terribly confusing, you realize. I haven't looked at the codebases of the package, but my mental model right now is a bit lost as to what package does what.

Reply:
It's not that confusing if you're coming to it fresh (which most people are).  It's really only a pain for the old hands who were around when we called the thing `distributed` on its own.